srcpkgs/pantheon-applications-menu/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'pantheon-applications-menu'
+pkgname=pantheon-applications-menu
+version=2.5.0
+revision=1
+wrksrc="applications-menu-${version}"
+build_style=meson
+configure_args="-Dwith-unity=false"
+hostmakedepends="cmake pkg-config glib-devel"
+makedepends="gnome-menus-devel granite-devel libsoup-devel AppStream-devel json-glib-devel zeitgeist-devel
+ switchboard-devel gtk+3-devel vala-devel libgee08-devel wingpanel-devel"
```

Comment:
A lot of these dependencies should not be necessary. wingpanel-devel should pull in granite-devel, and granite-devel always pulls in gtk and libgee. Plus the same issue with cmake and and vala-devel -- these should not be there. Please just use the commit from #20429 as-is.